packages/@aws-cdk/aws-iotevents/lib/detector-model.ts

```diff
@@ -16,6 +16,22 @@ export interface IDetectorModel extends IResource {
   readonly detectorModelName: string;
 }
 
+/**
+ * Information about the order in which events are evaluated and how actions are executed.
+ */
+export enum EvaluationMethod {
```

Comment:
Is this name too generic? Should this be `EventEvaluationMethod`, or `StateEvaluationMethod`, or `DetectorActionEvaluationMethod`, or something?

Actually, can we even kill the "Method" suffix here? I don't think it adds much, and just makes the name longer.

Reply:
`EventEvaluation` is good I think!